### PR TITLE
updates to compile on constance:

### PR DESCRIPTION
**Pull request by cmkaul**
These setting worked to compile nalu-wind on PNNL-Constance.

### CODE REVIEW

File: etc/spack/pnnl-constance/copy.packages.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
+packages:
```
Comment (sayerhs) [inline etc/spack/pnnl-constance/copy.packages.yaml]: Can you please delete these `copy.*.yaml` files before we merge the commit?

File: etc/spack/pnnl-constance/packages.yaml [hunk 2]
```diff
@@ -15,13 +15,13 @@ packages:
   openmpi:
     version: [1.10.1]
     buildable: false
-    modules:
-      openmpi@1.10.1: openmpi/1.10.1
+    paths:
```
Comment (sayerhs) [inline etc/spack/pnnl-constance/packages.yaml]: Curious. Any specific reason to prefer `paths` instead of `modules`?

Comment (cmkaul) [reply etc/spack/pnnl-constance/packages.yaml]: Simply what I found would work. I didn't spend the time to figure out why `modules` didn't work, but just took what seemed to be the route of least resistance.

Comment (sayerhs) [reply etc/spack/pnnl-constance/packages.yaml]: Thanks, that's what I was curious about.

**Pull request merged**